MangoPay/KycDocumentRefusedReasonType.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace MangoPay;
+
+/**
+ * Kyc Refused Reason Type
+ */
+class KycDocumentRefusedReasonType
+{
+    const DocumentUnreadable = 'DOCUMENT_UNREADABLE';
```

Comment:
In my opinion, const should be expressed in uppercase with underscores. This is however a purely checkstyle opinion. I'll leave the core-devs decide.

Reply:
Agree, but I followed the coding standard featured in this library.

Reply:
@iammichiel @gaetan-petit Good point, this definitely should be. I'll keep that in mind for some next major update !